### PR TITLE
l10n: Add dots

### PR DESCRIPTION
**Pull request by Valdnet**

### CODE REVIEW

**Comment by kesselb** [comment]
Mind to explain this change? I checked some other apps and summary and description does not end with a dot.

**Comment by Valdnet** [comment]
@kesselb There are complex sentences in the application description that require correct punctuation. They can contain various signs such as: semicolon, colon, comma etc. And it must contain a dot at the end of the sentence.

> I checked some other apps and summary and description does not end with a dot.
They do not appear in some application descriptions because they are often duplicated errors by developers.

I am trying to improve to ensure consistency for the whole Nextcloud. Only in this way, when order and compliance with the spelling is certain, the Nextcloud instance will gain even more popularity among users.

**Review by kesselb — APPROVED** [review]

**Comment by nickvergessen** [comment]
Too late, thuraday was string freeze, so translators can finish their work for Nextcloud 20. Will have to wait until 20 is out.

File: apps/contactsinteraction/appinfo/info.xml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 	  xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
 	<id>contactsinteraction</id>
 	<name>Contacts Interaction</name>
-	<summary>Manages interaction between users and contacts</summary>
-	<description>Collect data about user and contacts interactions and provide an address book for the data</description>
+	<summary>Manages interaction between users and contacts.</summary>
```
Comment (jancborchardt) [inline apps/contactsinteraction/appinfo/info.xml:6]: The summary is more like a short tagline and not a sentence. It should not have a dot at the end.

Comment (ChristophWurst) [reply apps/contactsinteraction/appinfo/info.xml:6]: I also often struggle with this. Can we have guidelines on these things in our dev or design docs?

Comment (Valdnet) [reply apps/contactsinteraction/appinfo/info.xml:6]: You have to take into account other punctuations that may appear in the summary. The slogan will look like an unfinished sentence.

**Review by nickvergessen — CHANGES_REQUESTED** [review]
Too late, 17th september was string freeze, so translators can finish their work for Nextcloud 20. Will have to wait until 20 is out.

**Pull request closed without merging**